### PR TITLE
Add team label

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/27145

## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/app-operator/templates/network-policy.yaml [hunk 1]
```diff
@@ -1,20 +0,0 @@
-kind: NetworkPolicy
```
Comment (QuentinBisson) [inline helm/app-operator/templates/network-policy.yaml:1]: Are you sure you want to remove the network policy?

**Comment by pipo02mix** [comment]
I only renamed the files to be aligned across repos, checking why it marks as deleted

File: helm/app-operator/templates/network-policy.yaml (hunk 1 above)
Comment (pipo02mix) [reply helm/app-operator/templates/network-policy.yaml:1]: I only renamed the files to be aligned across repos, checking why it marks as deleted

**Comment by pipo02mix** [comment]
super weird my local copy was not updating the changes from upstream I had to clone it again 🤷

**Comment by pipo02mix** [comment]
should we make this `false` by default?

**Comment by QuentinBisson** [comment]
@pipo02mix it's because the app operator is using a podmonitor and not a servicemonitor.

The setup is done here https://github.com/giantswarm/app-operator/blob/82531c12094078814174681b362712e71fd6674a/integration/setup/setup.sh#L7

File: .nancy-ignore [hunk 2]
```diff
@@ -1,82 +1,76 @@
...
+CVE-2023-28642 until=2023-07-30
 
 # pkg:golang/github.com/gin-gonic/gin@v1.9.0
-CVE-2023-29401 until=2023-06-30
+CVE-2023-29401 until=2023-07-30
 
 # golang/google.golang.org/grpc@v1.52.0
-CVE-2023-32731 until=2023-06-30
\ No newline at end of file
+CVE-2023-32731 until=2023-07-30
```
Comment (QuentinBisson) [inline .nancy-ignore:76]:
```suggestion
CVE-2023-32731 until=2023-07-30

```

**Review by mproffitt — APPROVED** [review]

**Pull request merged**